lms/djangoapps/bulk_email/tasks.py

```diff
@@ -522,8 +563,24 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
             # Pop the user that was emailed off the end of the list only once they have
             # successfully been processed.  (That way, if there were a failure that
             # needed to be retried, the user is still on the list.)
+            try:
+                recipients_info[email] += 1
+            except KeyError:
+                recipients_info[email] = 1
+
             to_list.pop()
 
+        log.info("Total Successful Recipients: %s, Failed Recipients: %s",
+                 total_recipients_successful, total_recipients_failed)
+        duplicate_recipients = ["{0} ({1})".format(email, repetition)
+                                for email, repetition in recipients_info.most_common() if repetition > 1]
```

Comment:
I'm guessing we'd want to show all the duplicate recipients, not just the most common. It would also be great to log how many emails they ended up getting.